src/lib/log.c

```diff
@@ -151,6 +193,16 @@ void log_info(const char *fmt, ...)
 }
 
 
+void log_warn(const char *fmt, ...)
+{
+	va_list args;
+
+	va_start(args, fmt);
+	log_vlog(ASL_LEVEL_WARNING, fmt, args);
+	va_end(args);
+}
```

Comment:
this seems to have strayed into a different patch to the addition of the other `log_{debug,info,err}` functions.

Reply:
I've moved it there, thanks.